### DISCUSSION TITLE
Traktor leert nicht den Mähdrescher

### DISCUSSION THREAD

**Discussion by Andro67** [Courseplay/courseplay#7675] [Q&A] [answered]
Hallo liebe Farmer,
Ich wunsche euch alles Gute zum Neues Jahr ...  *thumbsup*  :biggrin2: 
Ich habe eine Schwieriges problem, ich benutze das Courseplay für den Mähdrescher und Autodrive für den Traktor aber auf meine Ravenport und jetzt auch auf der Ravensberg Map geht den Traktor nicht zum Mähdrescher um in zu leeren, das funktionniert einwandfrei auf der Felsbrunn Map. 
Mein Log : [url]https://forum.giants-software.com/download/file.php?id=17877[/url]

**Comment by Tensuko** [comment, marked as answer]
Liegt nicht an der Map.
Sind Einstellungen, guck dir am besten Tutorials von Mario Hirschfeld auf YouTube an.

**Comment by Andro67** [comment]
Hi,
Die einstellungen sind wie auf der Felsbrunn map ich habe alles kopiert
aber es geht nicht. Auf der Ravensberg map hat es 2 mal fonktionniert mit
feld 5 und 14, dann habe ich noch feld 23 gekauft und dann ging plötzlich
nicht mehr. Das ist mittel eine game passiert ohne nichts zu ändern.
Danke für deine hilfe 🙂

**Comment by Andro67** [comment]
Hallo,
Du hattest recht, es war eine Einstellung aber im Autodrive der nicht
eingeschaltet ist wenn der Mähdrescher mit dem Courseplay läuft, hier drin
hatte ich nicht geguckt, ich hatte nicht gedacht dass dies  beeinflussen
kann.

Le mar. 11 janv. 2022 à 13:15, Fabrice KOEHREN ***@***.***> a
écrit :
